otherlibs/systhreads/thread.mli

```diff
@@ -58,49 +62,59 @@ val join : t -> unit
 (** [join th] suspends the execution of the calling thread
    until the thread [th] has terminated. *)
 
+val yield : unit -> unit
+(** Re-schedule the calling thread without suspending it.
+   This function can be used to give scheduling hints,
+   telling the scheduler that now is a good time to
+   switch to other threads. *)
+
+(** {1 Waiting for file descriptors or processes} *)
+
+(** The functions below are leftovers from an earlier, VM-based threading
+    system.  The [Unix] module provides equivalent functionality, in
```

Comment:
Small nitpick: It seems better to use a  reference, `{!Unix}`, in order to link to the Unix module page.

Reply:
I enthusiastically support the idea of putting cross-references.  See commit ab09ec418 .